2-1-6.py

```diff
@@ -0,0 +1,18 @@
+numbers = int(input())
+s_num = str(numbers)
+if len(s_num) == 5:
+    s_num = s_num[-1::-1]
+    for i in s_num:
```

Comment:
you can also traverse a string backward like this:
```
for i in reversed(s_num)
```

Reply:
Thank you teacher! This is a cool skill!